src/CNN/tools/Preprocessing.py

```diff
@@ -196,20 +196,24 @@ def getLabels(complex_data_path, index_path='Data/index/INDEX_refined_data.2016'
         if complexnames is None:
             complexnames = Preprocessing.getComplexDirNames(complex_data_path)
         with open(index_path) as f:
-            reader = csv.reader(f, delimiter='\t')
+            reader = csv.reader(f, delimiter=' ')
             data = [(col1)
                     for col1 in reader]
-        datas = []
-        for i in range(len(data)):
-            datas.append(data[i][0].split(' '))
+            data = [[i for i in d if i != ''] for d in data]
+#         datas = []
+#         for i in range(len(data)):
+#             datas.append(data[i][0].split(' '))
+        datas = [d for d in data if d[0] != '#']
         labels = []
-        complexes = list(np.array(datas)[:, 0])
+        complexes = [i[0] for i in datas]
+        #complexes = list(np.array(datas)[:, 0])
         for i in range(len(complexnames)):
             index = complexes.index(complexnames[i])
-            label = Preprocessing.extractKValue(datas[index])
+            label = float(datas[index][3])
+            #label = Preprocessing.extractKValue(datas[index])
             labels.append(label)
-        for i in range(len(labels)):
-            labels[i] = -np.log10(labels[i])
+        #for i in range(len(labels)):
+        #    labels[i] = -np.log10(labels[i])
         return labels
 
```

Comment:
For this change, there is already a second method, which reads the value and do not calc it. Is faster but not that precice